src/data/modifiers.txt

```diff
@@ -4621,7 +4621,6 @@ Familiar	Exotic Parrot	Hot Resistance: [floor((W+19)/20)], Cold Resistance: [flo
 Familiar	Happy Medium	Initiative: [W]
 Familiar	Hobo Monkey	Leprechaun: 1.25
 Familiar	Jumpsuited Hound Dog	Fairy: 1.25, Combat Rate: [min(5,floor(W/6))]
-Familiar	Jill-of-All-Trades	Fairy: [1+(0.5*equipped(LED candle)*pref(ledCandleMode,disco))], Leprechaun: [1+(0.5*equipped(LED candle)*pref(ledCandleMode,ultraviolet))], Volleyball Effectiveness: [1+(0.5*equipped(LED candle)*pref(ledCandleMode,reading))], Sombrero Effectiveness: [1+(0.5*equipped(LED candle)*pref(ledCandleMode,reading))]
```

Comment:
Is it correct that Jill is deleted completely?

Reply:
yes. By the attributes, it should default to 1.0 fairy / lep / volley / sombrero.

If it doesn't, report a bug.